dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -491,6 +491,14 @@ def select_dropdown(element, option_text, load)
   page_load(load) {element.click}
 end
 
+When /^I click "([^"]*)" if it is visible$/ do |selector|
```

Comment:
Note we have a step `When /^I click selector "([^"]*)" if it exists$/` that is very similar to this, but uses the built in [HTMLElement click() method](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click), rather than Selenium. I'm not 100% sure why this doesn't work.